src/Conductors/ChecksRole.php

```diff
@@ -47,6 +47,17 @@ public function a($role)
     }
 
     /**
+     * Check if the user doesn't have any of the given roles.
+     *
+     * @param  string  $role
+     * @return bool
+     */
+    public function notA($role)
```

Comment:
This is just an alias of `a()`, so it's just passing the same arguments. `a()` is where it'll do `func_get_args()`. Trying to be as DRY as possible. You can see Laravel's example in a [Collection's](https://github.com/laravel/framework/blob/5.1/src/Illuminate/Support/Collection.php) `lists()` function call, which is an alias to `pluck()`.

```
    /**
     * Get the values of a given key.
     *
     * @param  string  $value
     * @param  string|null  $key
     * @return static
     */
    public function pluck($value, $key = null)
    {
        return new static(Arr::pluck($this->items, $value, $key));
    }
    /**
     * Alias for the "pluck" method.
     *
     * @param  string  $value
     * @param  string|null  $key
     * @return static
     */
    public function lists($value, $key = null)
    {
        return $this->pluck($value, $key);
    }
```

As you can see it's just calling its own function passing along the arguments. Same logic applies here. No sense in duplicating the logic within `an()` if it's going to do the exact same thing of `a()`. Just call `a()` (or `notA` and `notB`). Applies to the other comments you made below.